libs/designer/src/lib/ui/panel/connectionsPanel/createConnection/custom/acasessionConnector.tsx

```diff
@@ -34,7 +34,7 @@ export const ACASessionConnector = (props: ConnectionParameterProps) => {
     if (isFetchingBuiltInRoleDefinitions || !Array.isArray(builltInRoleDefinitions)) {
       return undefined;
     }
-    return builltInRoleDefinitions?.find((role: any) => role?.properties?.roleName === 'Azure ContainerApps Session Executor')?.name;
+    return builltInRoleDefinitions?.find((role: any) => role?.properties?.roleName === 'Azure Container Apps Session Executor')?.name;
```

Comment:
This is a role name, not a string being used, changing this would make it so that we no longer filter based on the correct role